database/seeders/EmulatorsTableSeeder.php

```diff
@@ -25,7 +25,7 @@ public function run(): void
                 'name' => $emulatorRelease['name'],
                 'handle' => $emulatorRelease['handle'],
                 'active' => $emulatorRelease['active'],
-                'link' => $emulatorRelease['link'],
+                'link' => $emulatorRelease['link'] ?? null,
```

Comment:
Fixes a null reference seeding the database with the most recent releases.dist.php.